django/core/jinja2/core/jobs/list.jinja

```diff
@@ -1,6 +1,22 @@
 {% extends "search_layout.jinja" %}
 {% from "common.jinja" import list_page, paginator, breadcrumb, member_profile_href, search_tag_href, search_bar %}
 
+{% block ogp_tags %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://www.comses.net/jobs/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Community Jobs">
+    <meta property="og:description" content="View all jobs related to CoMSES">
+    <meta property="og:image" content="https://complexity.asu.edu/sites/default/files/comses.jpg">
```

Comment:
should we create custom images for comses jobs / events / codebases and add them to static? might be a nice UX touch